js/form-validity.js

```diff
@@ -49,8 +49,54 @@ const onRoomsChange = () => {
   if (!(roomsToCapacities[roomNumber].includes(capacityNumber))) {
     formCapacity.setCustomValidity('Количество гостей больше чем комнат');
   }
+  else {
+    (formCapacity.setCustomValidity(''));
+  }
 };
 formCapacity.addEventListener('input',() => {
   onRoomsChange();
   formCapacity.reportValidity();
 });
+
+// Устанавливаем валидность для поля "Тип жилья" и "Цена за ночь"
+const formHouseType = document.querySelector('#type');
+const formHousePrice = document.querySelector('#price');
+const typeToPrice = {
+  bungalow : [0],
```

Comment:
```suggestion
  bungalow : 0,
```
тут массив не нужен)